identity-server/Directory.Build.props

```diff
@@ -29,7 +29,7 @@
 
     <PropertyGroup>
         <MinVerAutoIncrement>minor</MinVerAutoIncrement>
-        <MinVerPrefix>is-</MinVerPrefix>
```

Comment:
The main branch was actually temporarily wrong while we were getting 7.1 built, but we were building out of the release branch, so this configuration didn't get used.